cocos/ui/UITextField.cpp

```diff
@@ -46,6 +46,20 @@ UICCTextField::~UICCTextField()
 {
 }
 
+UICCTextField * UICCTextField::create()
+{
+    UICCTextField *pRet = new (std::nothrow) UICCTextField();
```

Comment:
pRet -> ret
we don't use Hungarian Notation now. Old codes need to be modified when meet it.